src/components/Gouvernance/Gouvernance.test.tsx

```diff
@@ -492,7 +494,66 @@ describe('gouvernance', () => {
     const rowsBody = within(body).getAllByRole('row')
     const columns1Body = within(rowsBody[0]).getAllByRole('cell')
     expect(columns1Body).toHaveLength(3)
-    expect(columns1Body[1].textContent).toBe('Comité stratégique en attente de planification')
+    expect(columns1Body[1].textContent).toBe('Comité stratégique')
+    expect(columns1Body[2].textContent).toBe('Semestriel')
+  })
+
+  it('quand j’affiche une gouvernance avec un comité dont la date est dans le passé, alors le comité est affiché sans date', () => {
+    // GIVEN
+    const gouvernanceViewModel = gouvernancePresenter(gouvernanceReadModelFactory({
+      comites: [
+        {
+          commentaire: 'commentaire',
+          date: new Date('2020-09-06'),
+          nom: 'Comité stratégique 1',
+          periodicite: 'Semestriel',
+          type: 'stratégique',
+        },
+      ],
+    }))
+
+    // WHEN
+    render(<Gouvernance gouvernanceViewModel={gouvernanceViewModel} />)
+
+    // THEN
+    const comites = screen.getByRole('table', { name: 'Comités' })
+    const rowsGroup = within(comites).getAllByRole('rowgroup')
+    const body = rowsGroup[1]
+    const rowsBody = within(body).getAllByRole('row')
+    const columns1Body = within(rowsBody[0]).getAllByRole('cell')
+    expect(columns1Body).toHaveLength(3)
+    expect(columns1Body[1].textContent).toBe('Comité stratégique')
+    expect(columns1Body[2].textContent).toBe('Semestriel')
+  })
+
+  it('quand j’affiche une gouvernance avec un comité dont la date est le jour même ou dans le futur, alors le comité est affiché avec sa date', () => {
+    // GIVEN
+    const dateJourJ = new Date('2024-09-06')
+    vi.setSystemTime(dateJourJ)
+
+    const gouvernanceViewModel = gouvernancePresenter(gouvernanceReadModelFactory({
+      comites: [
+        {
+          commentaire: 'commentaire',
+          date: new Date('2024-09-06'),
+          nom: 'Comité stratégique 1',
+          periodicite: 'Semestriel',
+          type: 'stratégique',
+        },
+      ],
+    }))
+
+    // WHEN
+    render(<Gouvernance gouvernanceViewModel={gouvernanceViewModel} />)
+
+    // THEN
+    const comites = screen.getByRole('table', { name: 'Comités' })
+    const rowsGroup = within(comites).getAllByRole('rowgroup')
+    const body = rowsGroup[1]
+    const rowsBody = within(body).getAllByRole('row')
+    const columns1Body = within(rowsBody[0]).getAllByRole('cell')
+    expect(columns1Body).toHaveLength(3)
+    expect(columns1Body[1].textContent).toBe('Comité stratégique : 06/09/2024')
```

Comment:
```suggestion
    const comites = screen.getByRole('table', { name: 'Comités' })
    const columns1Body = within(comites).getAllByRole('cell')
    expect(columns1Body[1].textContent).toBe('Comité stratégique : 06/09/2024')
```
Reproduis ça pour tes autres tests aussi.
Vu qu'on a déjà testé tout ça avant, pas la peine de le refaire car c'est déjà fait et ça fait du "bruit pour rien", les tests UI sont déjà assez gros donc autant les réduire dès qu'on peut.